homeassistant/components/yolink/binary_sensor.py

```diff
@@ -25,20 +31,37 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
     """YoLink BinarySensorEntityDescription."""
 
     exists_fn: Callable[[YoLinkDevice], bool] = lambda _: True
+    key: str = "state"
```

Comment:
The descriptions shouldn't all have the same value for the `key` attribute. The key value should be unique among the descriptions in the platform.

Reply:
Since our device api design, most binary sensors `key` attribute is `state`, So I give `key` attribute a default value，For special sensors, I will override this `key` attribute.

Reply:
There's no guard for adding more than one sensor per device and we use the sensor description key in the unique_id. So the key must be unique across the descriptions.

We can add another description attribute for looking up the api value.

Reply:
yolink binary sensor will not have more than one description for per device, but I think I still need to fix this issue.

Reply:
I will create a new PR to solve this, Thanks